iceoryx_posh/source/runtime/ipc_interface_base.cpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 by Apex.AI Inc. All rights reserved.
```

Comment:
wrong year

Reply:
This is [not](https://github.com/eclipse-iceoryx/iceoryx/pull/538#discussion_r569483365) a new file. These are the old values:
```
iceoryx_posh/source/runtime/message_queue_interface.cpp
---
// Copyright (c) 2019, 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```
and
```
iceoryx_posh/internal/runtime/message_queue_interface.hpp
---
// Copyright (c) 2019, 2021 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Reply:
But this is a completely new file according to the git history and also completely new code. 
@mossmaurice I agree with you but then we have to use `git mv` from `mq_interface` to this file to restore the history and then I would leave it like this otherwise the git history proofs you wrong and I could add random years and copyrights to every file with the argument yeah but it is code copied from somewhere else and the history was lost.

Reply:
@mossmaurice Additionally, you touched the code `IpcMessage` can be found in here and was introduced in this pull request.

Reply:
Alrighty, adapted. `git mv` only works 1:1 and not 1:m unfortunately.